internal/controller/docs/cluster_provider.go

```diff
@@ -8,7 +8,7 @@ import (
 	"time"
 
 	"github.com/kyma-project/helm-broker/internal"
-	"github.com/kyma-project/kyma/components/cms-controller-manager/pkg/apis/cms/v1alpha1"
+	"github.com/kyma-project/helm-broker/pkg/apis/rafter/v1beta1"
```

Comment:
Why you don't use existing types from [here](https://github.com/kyma-project/rafter/tree/master/pkg/apis/rafter/v1beta1) ?

Reply:
Rafter and Helm-Broker have the other version of some dependencies. Especially rafter has `api-machinery` in version 0.2.0 but helm-broker in 0.1.12 and it generates a lot of conflicts. After a discussion with the `gopher` team, we decided to copy the rafters client into the `pkg` folder in helm-broker.

Reply:
please add a comment to all files why they are copied and when it could be removed (e.g. files could be removed when `libX` has version >= 1.0)

Reply:
Good idea. Thanks!